unity/MobiledgeXRestSample/Assets/Plugins/iOS/PlatformIntegration.m

```diff
@@ -1,5 +1,9 @@
 /**
+<<<<<<< HEAD
```

Comment:
Looks like you have some merge conflicts? Or maybe residual generated code from git from previous merge conflicts?